tests/testthat/test_OptimizerNLoptr.R

```diff
@@ -0,0 +1,9 @@
+context("OptimizerNLoptr")
+
+test_that("OptimizerNLoptr", {
+  param_set = list(x0 = 1)
+  z = test_optimizer(OptimizerNLoptr$new(), param_set, n_dim = 1,
+                     term_evals = 100L, real_evals = NA)
+  expect_class(z$optimizer, "OptimizerNLoptr")
+  expect_output(print(z$optimizer), "OptimizerNLoptr")
```

Comment:
Since you removed the `expect_data_table(archive$data())` in `test_optimizer()` for this test. What are you expecting here?